routers/web/user/profile.go

```diff
@@ -90,6 +91,46 @@ func Profile(ctx *context.Context) {
 		ctx.Data["RenderedDescription"] = content
 	}
 
+	repo, err := repo_model.GetRepositoryByName(ctx.ContextUser.ID, ".profile")
+	if err == nil && !repo.IsEmpty {
+		gitRepo, err := git.OpenRepository(ctx, repo.RepoPath())
+		if err != nil {
+			ctx.ServerError("OpenRepository", err)
```

Comment:
```suggestion
			ctx.ServerError("OpenRepository", err)
			return
```
`ServerError` does not stop execution.

Reply:
That's still not fixed.

Reply:
If there's an issue opening the repo, why do I want execution to stop? Instead wouldn't it make more sense to print an error and just not continue processing a profile README? The code produces an expected result currently.

Reply:
I talk about all the other `ServerError` calls. If you call that method you can't send a regular response afterwards. You need to ignore/log all errors or stop executing.

Reply:
I guess I am still confused. Are you suggesting I remove all ServerError calls?

Reply:
No, just add a `return` if it's a non-recoverable error (and that's true for every `ServerError` call you have at the moment). If it's an expected/recoverable error just do nothing (and/or log it).

Reply:
Implemented